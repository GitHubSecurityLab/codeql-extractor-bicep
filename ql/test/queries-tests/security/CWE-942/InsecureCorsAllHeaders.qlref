security/CWE-942/InsecureCorsAllHeaders.ql