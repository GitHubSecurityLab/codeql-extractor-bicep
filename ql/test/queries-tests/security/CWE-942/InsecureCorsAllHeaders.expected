| app.bicep:42:27:42:33 | Array | CORS policy allows all headers ("*"), which can increase attack surface. |
